sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -185,6 +185,19 @@ export type DataFeedGranularity =
       granularityType: "Custom";
       customGranularityValue: number;
     };
+/**
+ * The flattened response to a REST call.
+ * Contains the underlying HttpOperationResponse as well as
+ * the merged properties of the parsedBody, parsedHeaders, etc.
+ */
+export interface RestResponse {
+  /**
+   * The underlying HTTP response containing both raw and deserialized response data.
+   */
+  _response: FullOperationResponse;
```

Comment:
`FullOperationResponse` is not compatible with the old `HttpOperationResponse`.  Looks that we'd need more centralized work.

```ts
export interface HttpOperationResponse extends HttpResponse {
```

```ts
export interface FullOperationResponse extends PipelineResponse {
```